### ISSUE TITLE
Links on Profile pages are broken

### ISSUE THREAD

**Issue by lazyguru** [kalenjordan/magehero#129]
The links for a user's Twitter, GitHub, etc.. are broken when viewing their "profile" page.  Eg. Twitter link is wrong here: http://magehero.com/mhauri  but works fine here: http://magehero.com/search?query=hauri

**Comment by kalenjordan** [comment]
Thanks.  Must be related to https://github.com/kalenjordan/magehero/pull/125/files @Narno

**Comment by Narno** [comment]
Hum, I know why:
```
{% if user.twitterUsername() %}
    <a href="https://twitter.com/{{ developer.twitterUsername() }}" target="_blank"><i class="fa fa-twitter tooltip" title="Twitter"></i></a>
{% endif %}
```
```user``` <> ```developer```

I can make a patch quickly. Sorry for that.

**Comment by kalenjordan** [comment]
Thanks!

On Mon, Jun 1, 2015 at 3:23 PM, Arnaud Ligny <notifications@github.com>
wrote:

> Hum, I know why:
>
> {% if user.twitterUsername() %}
>     <a href="https://twitter.com/{{ developer.twitterUsername() }}" target="_blank"><i class="fa fa-twitter tooltip" title="Twitter"></i></a>
> {% endif %}
>
> user <> developer
>
> I can make a patch quickly. Sorry for that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kalenjordan/magehero/issues/129#issuecomment-107734874>
> .
>



-- 
Kalen Jordan
Founder, MageMail

Listen to MageTalk - The Magento Podcast <http://magetalk.com/> | MageMail
- The Triggered Email App for Magento
<http://magemail.co/?utm_medium=email&utm_source=signature&utm_campaign=signature>

**Comment by beejhuff** [comment]
I saw this earlier this morning and hacked out an update to make sure the references to developer.<attribute> were replaced with user.<attribute>...submitted the pull request #131 a few minutes ago...

Decided to see if there were any other open issues I could assist with when I noticed this thread...hope I didn't inconvenience anyone....I'll be sure to check the issues list BEFORE I submit a pull request next time ;)

**Comment by kalenjordan** [comment]
Thanks Bryan - appreciate the effort.  I just merged and deployed #130.

**Comment by beejhuff** [comment]
Please Kalen, only my Mom and my ex call me Bryan and it's never a good thing when they do ;)  BJ is fine.  I shot you an invite to check out a thread I was participating in with @razbakov and @flyingmana over in gitter when I saw you hadn't setup an account with it them yet. I'm sure you're bus.y but it's a pretty neat little app somewhere between slack / hipchat, these comment forums and IRC.  At any rate, I think there's a few topics over there in their CI chat and one I just wrote up on the recent security vulnerabilities that could benefit from your input...
